src/Components/SingleProductPage/ProductImageSection/ProductImageSection.jsx

```diff
@@ -0,0 +1,45 @@
+import { useWishlist } from "../../../Context";
```

Comment:
You can path alias (just something to know) bcz if you have a larger hierarchy then it would be difficult to use a relative path.

Reply:
It is importing the path automatically after providing the component's name. So I guess there won't be any issue.